### ISSUE TITLE
redis path changed

### ISSUE THREAD

**Issue by jesseky** [gin-contrib/sessions#66]
[github.com/garyburd/redigo](https://github.com/garyburd/redigo) 
has been moved to 
[github.com/gomodule/redigo](https://github.com/gomodule/redigo)

**Comment by SilverCory** [comment]
Yes, also make sure you're using the master branch of github.com/boj/redistore
